docs/topics/kotlin-ide.md

```diff
@@ -15,6 +15,22 @@ Kotlin plugin is bundled with each IntelliJ IDEA release.
 
 Read more about IntelliJ IDEA in the [official documentation](https://www.jetbrains.com/help/idea/discover-intellij-idea.html).
 
+## Fleet
+
+> JetBrains Fleet is currently in Public Preview, and at this stage is free to use.
+{type="note"}
+
+[JetBrains Fleet](https://www.jetbrains.com/fleet/) is a polyglot IDE and code editor that provides advanced support for Kotlin
+and a streamlined experience for Kotlin developers. You can use Fleet as a code editor to quickly make targeted edits,
+or turn on Smart Mode and transform it into a full-fledged IDE with code intelligence features.
+
+On top of that, with Fleet, you can quickly open and run Kotlin Multiplatform projects targeting the Android, iOS, and desktop
```

Comment:
Found this too heavy.
Can we omit "on top of that"? So the sentence will be a bit more compact.
Also, the part in the brackets could be a separate sentence (or "because Smart Mode selects the ...").
WDYT?